_src/get-started.html

```diff
@@ -111,8 +111,7 @@
 
     <!-- Step 4 Code -->
     <div class="col col-12">
-      <p>Download systemml-0.11.0-incubating.zip and unzip it to a location of your choice.
-          <a>https://systemml.apache.org/download.html</a>.<br>
+      <p>Download <a href="http://mirror.atlanta.delimiter.com/pub/apache/incubator/systemml/0.10.0-incubating/systemml-0.10.0-incubating.zip" target="_blank">systemml-0.10.0-incubating.zip</a> and unzip it to a location of your choice.<br>
```

Comment:
@jazares Could we change the download link to the following URL: http://www.apache.org/dyn/closer.cgi/pub/apache/incubator/systemml/0.10.0-incubating/systemml-0.10.0-incubating.zip